### PR TITLE
Veli Javascript2 Week1

### PR DESCRIPTION
**Pull request by veliataseven**

### CODE REVIEW

File: Week1/homework/app.js [hunk 1]
```diff
@@ -1,11 +1,164 @@
...
+  }
+}
+
+// makeList(bookTitles);
+
+// ####################################  1.4  ###############################
+
+const books = {
+  the_godfather: {
+    id: '',
```
Comment (wilgert) [inline Week1/homework/app.js]: Why do you need an empty `id` property here?

File: Week1/homework/app.js [hunk 2]
```diff
@@ -1,11 +1,164 @@
...
+  'anna_karenina',
+  'war_and_peace',
+];
+
+// console.log(bookTitles);
+
+// ####################################  1.3  ###############################
+
+// eslint-disable-next-line no-unused-vars
+function makeList(arr) {
```
Comment (wilgert) [inline Week1/homework/app.js]: Nice function!

File: Week1/homework/app.js [hunk 3]
```diff
@@ -1,11 +1,164 @@
...
+  nineteen_eighty_four: './img/nineteen_eighty_four.jpg',
+  the_lord_of_the_rings: './img/the_lord_of_the_rings.jpg',
+  a_tale_of_two_cities: './img/a_tale_of_two_cities.jpg',
+  anna_karenina: './img/anna_karenina.jpg',
+  war_and_peace: './img/war_and_peace.jpg',
+};
+
+// ####################################  1.8  ###############################
+
+console.log(Object.keys(imgPath));
```
Comment (wilgert) [inline Week1/homework/app.js]: Please try to not push `console.log` statements like this one to GitHub. It's okay to use them while developing, but it's best to keep them out of the definitive code.

**Comment by fedefu** [comment]
Thanks for the feedback @wilgert . @veliataseven can you please implement suggestions? Thanks!

**Comment by fedefu** [comment]
My bad then! new to this :P Good job Veli!

**Pull request closed without merging**